packages/typespec-ts/src/lib.ts

```diff
@@ -77,7 +81,8 @@ export const RLCOptionsSchema: JSONSchemaType<RLCOptions> = {
     enableOperationGroup: { type: "boolean", nullable: true },
     enableModelNamespace: { type: "boolean", nullable: true },
     hierarchyClient: { type: "boolean", nullable: true },
-    branded: { type: "boolean", nullable: true, default: true }
+    branded: { type: "boolean", nullable: true, default: true },
```

Comment:
maybe we should remove the default logic for branded?

Reply:
Good catch, thanks!